sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIOTest.java

```diff
@@ -424,66 +447,6 @@ private static KV<ByteString, Iterable<Mutation>> makeBadWrite(String key) {
     return KV.of(ByteString.copyFromUtf8(key), mutations);
   }
 
-  /** Tests that credentials are used from PipelineOptions if not supplied by BigtableOptions. */
-  @Test
```

Comment:
Just wondering the purpose of removing these unit tests, is it because the scenario is no longer applicable for the new client, or are there any breaking change / regression could be introduced?

Reply:
Good catch. The way we get BigtableService is different now so these tests are no longer applicable. I moved the tests to BigtableConfigTranslator which will test the same behavior, PTAL, thanks! https://github.com/apache/beam/pull/24015/commits/97e96e85d2aba1ac2e2e2499dc35c02444c3549b#diff-b53a99acd5a4cd30f4b76e8b94f7cd09df6f0fc196d949842d20bc845b48661c